docs/api.rst

```diff
@@ -2815,6 +2843,54 @@ AutoModerationActionExecution
 .. autoclass:: AutoModerationActionExecution()
     :members:
 
+Poll
+~~~~
+
+.. attributetable:: Poll
+
+.. autoclass:: Poll()
+   :members:
+
+PollMedia
+~~~~~~~~~
+
+.. attributetable:: PollMedia
+
+.. autoclass:: PollMedia()
+   :members:
+
+PollAnswer
+~~~~~~~~~~
+
+.. attributetable:: PollAnswer
+
+.. autoclass:: PollAnswer()
+   :members:
+
+PollCreateRequest
+~~~~~~~~~~~~~~~~~
+
+.. attributetable:: PollCreateRequest
+
+.. autoclass:: PollCreateRequest()
```

Comment:
Since the below example shows you constructing a `PollCreateRequest`, this shouldn't have the trailing `()` in order to show the signature in the docs. This shouldn't be included for any user-instantiated class.